src/lib/libone_utils.cpp

```diff
@@ -232,6 +234,30 @@ unsigned long getLength(const std::shared_ptr<librevenge::RVNGInputStream> input
   return getLength(input.get());
 }
 
+template <typename T>
+std::string int_to_hex(const T i)
+{
+  std::stringstream stream;
+
+  stream << std::hex
+         << std::setfill('0')
+         << std::setw(sizeof(T)*2)
```

Comment:
What does the `setw` do in this case? I understand it is to format the output string, but even when replacing `T` with one of the types below in my head, I'm having trouble :)

Reply:
`setw` sets the width of the output string. Predefining the width is necessary to refill the string with zeros if the hex number has leading zeros.
`std::hex` seems to neglect leading zeros.